src/main/java/com/conveyal/datatools/manager/jobs/DeploymentGisExportJob.java

```diff
@@ -29,15 +29,21 @@ public DeploymentGisExportJob(GisExportJob.ExportType exportType, Deployment dep
         try {
             File outDir = setupGisExport();
 
+            int feedVersionsSize = feedVersions.size();
+            int currentFeedVersion = 1;
             for (SummarizedFeedVersion feedVersion: feedVersions) {
+                int percentComplete = (int) (currentFeedVersion * 100.0f) / feedVersionsSize;
```

Comment:
Are there any java math util we can use to avoid this cast?

Reply:
How you like these apples: https://github.com/ibi-group/datatools-server/pull/471#:~:text=avoid%20int%20casting-,34cd337,-Merge%20state